contrib/test/run_test_vectors.sh

```diff
@@ -42,18 +48,20 @@ else
   exit 0
 fi
```

Comment:
I just reverted my changes copying from main. This way when we rebase this part doesn't create issues.

@ravyu-jump note that the elf is not really checked, but the `set -e` above will catch issues.

Reply:
yeah there is actually an issue with elf test vectors atm where we added new fields but didn't update the existing fixtures so I expect this test to fail. Working on updating the fixtures right now.